privacyidea/models.py

```diff
@@ -2654,7 +2662,7 @@ class Audit(MethodsMixin, db.Model):
     id = db.Column(db.Integer, Sequence("audit_seq"), primary_key=True)
     date = db.Column(db.DateTime)
```

Comment:
Don't we need to set a second precision here, too?
No, this is done by adding compiles decorator to the DateTime column type?